### PR TITLE
Added additional configuration option of postMessage

### PR DESCRIPTION
**Pull request by MarcelloDiSimone**
I added all available configuration options to the task that are available for Slacks postMessage API call. I also updated the README file to reflect the changes. JSDoc comment have been added to the script. 

The new configuration options are:
as_user, parse, link_names, attachments, unfurl_links, unfurl_media, con_url and con_emoji

This pull request is based on my prior PR #1 therefor I would recommend to pull #1 first.

### CODE REVIEW

**Comment by germanrcuriel** [comment]
I've merged #1. I will check this PR asap

**Comment by germanrcuriel** [comment]
~~Attachments are working for you? I mean... this grunt task relies in another package that I own already and it doesn't have the attachments resource implemented yet.~~
Nevermind, I got confused with other service :P

File: README.md [hunk 1]
```diff
@@ -58,6 +59,54 @@ Default: `Grunt.js`
...
+Type: `Boolean`
+Default: `false`
+
+Set to true if you want references to private/public channels or users to be automatically linked
+
+#### attachments
+Type: `Array`
+Default: `null`
+
+An list of attachements for your message. See https://api.slack.com/docs/attachments#message_formatting
```
Comment (germanrcuriel) [inline README.md:84]: Typo in `attachements`

**Comment by germanrcuriel** [comment]
All seems OK. I only want to mention that I will move the JSDoc comments to a markdown file after merging because it's the "common" practice in grunt tasks repos.

**Comment by MarcelloDiSimone** [comment]
Perfect! Didn't knew about the JSDoc convention, sorry.

**Comment by abbondanza** [comment]
@MarcelloDiSimone Will this be merged in soon? This would be extremely useful!

**Pull request merged**